addon/components/o-s-s/chip.hbs

```diff
@@ -1,4 +1,4 @@
-<div class={{this.computedClass}} ...attributes>
+<div class={{this.computedClass}} ...attributes {{on "click" this.stopPropagation}}>
```

Comment:
hmmmm ? 🤔 
Why ?

Reply:
Because when used in the PowerSelect it triggers the click outside + the only thing we want to monitor clicks on here is the cross icon